src/helpers/numbers.js

```diff
@@ -19,13 +19,24 @@ export const numberFormat = (
 ) => {
     const style = currency ? "currency" : "decimal";
 
+    const isCanadianLocale = ["en-CA", "fr-CA"].includes(locale);
```

Comment:
i think we should not have any custom local for Canada. 

Here, the issue in this ticket is user have set the locale to `en_US` (this will be in seller document) and we should show the currency in this locale instead of the current location locale.  Incase the seller document don't have locale set then you can use the current locale. 

As the seller set the locale to `en_US` and if opens the app from Canada, we should still show the currency in `en_US` format instead of `en_CA`. so i think no changes might be required in ui-kit, you just need to pass the correct locale from seller app.